docs/components/Page/Web.stories.tsx

```diff
@@ -68,10 +73,20 @@ WithActions.args = {
   ],
 };
 
-export const WithSubtitle = BasicTemplate.bind({});
-WithSubtitle.args = {
+export const WithIntro = BasicTemplate.bind({});
+WithIntro.args = {
   title: "Notifications",
   subtitle: "Notify me of all the work",
   intro:
     "Improve job completion rates, stop chasing payments, and boost your customer service by automatically communicating with your clients at key points before, during, and after a job. Read more about Notifications by visiting our [Help Center](https://help.getjobber.com/hc/en-us).",
+  externalIntroLinks: true,
+};
+
+export const WithAdditionalTitleFields = BasicTemplate.bind({});
```

Comment:
oooooh thank you!

Reply:
Would love to see this example with actions so I can check how it looks when the actions and the label wrap!

Reply:
Alternatively, if you can drop some examples of how that looks at small and large breakpoints on this comment, I can design review that way 😃

Reply:
![Screenshot 2024-06-20 at 11 21 19](https://github.com/GetJobber/atlantis/assets/6353164/7ecdea4f-9faf-447e-b9fc-6e87c7c70b9e)
![Screenshot 2024-06-20 at 11 21 11](https://github.com/GetJobber/atlantis/assets/6353164/cb3706df-12ad-4cf2-b673-ff94e977fa41)
![Screenshot 2024-06-20 at 11 20 59](https://github.com/GetJobber/atlantis/assets/6353164/4437861e-fb76-4a0f-94ee-5844c731adf0)
![Screenshot 2024-06-20 at 11 20 49](https://github.com/GetJobber/atlantis/assets/6353164/e87ed579-1e1e-4ab3-a896-017cf87584a3)
![Screenshot 2024-06-20 at 11 20 10](https://github.com/GetJobber/atlantis/assets/6353164/f8f9bae8-5ec3-4c08-a5c4-ea43b288bc5c)
![Screenshot 2024-06-20 at 11 20 01](https://github.com/GetJobber/atlantis/assets/6353164/0463ea54-e090-4548-9d51-0f3fbeff85d5)
![Screenshot 2024-06-20 at 11 19 51](https://github.com/GetJobber/atlantis/assets/6353164/493301aa-1e87-46f1-a251-05640c81822c)
![Screenshot 2024-06-20 at 11 19 44](https://github.com/GetJobber/atlantis/assets/6353164/9515c0e9-62b4-4b8f-93e0-aa18ad59a8b1)
![Screenshot 2024-06-20 at 11 19 32](https://github.com/GetJobber/atlantis/assets/6353164/6f011926-1196-4ce5-b49c-21819e92b680)

Reply:
Amazing, thanks @jlapstra !